packages/app-bridge/src/registries/commands/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './AssetViewer';
 export * from './CommandRegistry';
 export * from './NavigationManager';
 export * from './TemplateChooser';
+export * from './CreateNewPublication';
```

Comment:
Missing file

Reply:
Yep, added a command that aligns the new code structure.